packages/flex-plugin-e2e-tests/src/utils/pages/view/agent-desktop.ts

```diff
@@ -0,0 +1,30 @@
+import { ElementHandle, Page } from 'puppeteer';
+
+import { Base } from '../base';
```

Comment:
Sorry for the nitpick, but it looks like all the "views" are extending this "base" class, but the "base" class in the "pages" directory? What's the difference between view/ and page/? Can they merged into one directory? If not, then move the Base.ts to the view because only views are extending it

Reply:
Pages essentially describes the whole page object pattern as a whole hence a dedicated folder. I wanted to split entry point (App) from each individual view since it will contain each view vs view being used directly.
But good point on base, I will move it to views since it is being extended only by classes in the /views